packages/api/src/command/medical/facility/create-facility.ts

```diff
@@ -32,19 +33,25 @@ export const createFacility = async ({
     data,
   };
   validateCreate(input);
+  const facilityByNpi = await getFacilityByNpi({ cxId, npi: data.npi });
+  if (facilityByNpi) {
+    throw new BadRequestError(
+      `Found a matching facility along NPI - please add the corresponding Facility ID to the request to update`
```

Comment:
this message is a bit confusing to me, perhaps:

`Can't create a new facility with the same NPI as facility with ID: ${facilityByNpi.id} and name: ${facilityByNpi.name}`

Reply:
Done.